File: backend/host/Models/Nutrients.cs
using System.Text.Json.Serialization;
using host.DataBaseAccess;

namespace host.Models;

public class Nutrients : ITableDataBase
{
    public int? Id { get; set; }
    [JsonPropertyName("menu_item_id")] public int? MenuItemId { get; set; }
    public int? Kilocalories { get; set; }
    public int? Proteins { get; set; }
    public int? Fats { get; set; }
    public int? Carbohydrates { get; set; }

    public Nutrients(){}
    public Nutrients(int? id, int? kilocalories, int? proteins, int? fats, int? carbohydrates)
    {
        Id = id;
        Kilocalories = kilocalories;
        Proteins = proteins;
        Fats = fats;
        Carbohydrates = carbohydrates;
    }
}

File: frontend/Business/tg_business_bot/Models/Nutrients.py
from pydantic import BaseModel
from typing import Optional


class Nutrients(BaseModel):
    kilocalories: Optional[int]
    proteins: Optional[int]
    fats: Optional[int]
    carbohydrates: Optional[int]

    def __init__(self,
                 kilocalories: Optional[int] = None,
                 proteins: Optional[int] = None,
                 fats: Optional[int] = None,
                 carbohydrates: Optional[int] = None):
        super().__init__(
            kilocalories=kilocalories,
            proteins=proteins,
            fats=fats,
            carbohydrates=carbohydrates
        )